src/registerToolbar.ts

```diff
@@ -56,6 +51,38 @@ export default function registerToolbar() {
         type: ToolbarItemType.State,
     });
 
+    ToolbarItemFactory.register(CopyRegions, {
+        name: "copyRegions",
```

Comment:
Shouldn't the accelerators be hooked up here?

Reply:
Would you have accelerators on both the buttons and within the `Canvas` component?